Core/MIPS/ARM/ArmJit.cpp

```diff
@@ -46,7 +46,7 @@
 
 using namespace ArmJitConstants;
 
-static void DisassembleArm(const u8 *data, int size) {
+void DisassembleArm(const u8 *data, int size) {
```

Comment:
MSVC keeps telling me this is an unresolved external symbol although It's not, so I remove the static and it works.
Not sure why